static/networks/butterfly.json

```diff
@@ -1,25 +1,26 @@
 {
   "Name":"butterfly",
-  "Purpose": "For core implementers testing new code. (frequent resets and not recommended for most users) 512MiB, 32GiB, 64GiB Low",
+  "Summary": "For core implementers testing new code.",
+  "Description": "This network suffers frequent resets and is not recommended for most users",
   "Maintainer": "Protocol Labs",
   "Stability": "low",
   "Status": "online",
   "NetworkParameters":{
     "SupportedProofTypes":[
       {
-        "Name":"RegisteredSealProof_StackedDrg512MiBV1",
-        "Size":"536870912",
-        "RegisteredSealProof": 2
+	"Name":"RegisteredSealProof_StackedDrg512MiBV1",
+	"Size":"536870912",
+	"RegisteredSealProof": 2
       },
       {
-        "Name":"RegisteredSealProof_StackedDrg32GiBV1",
-        "Size":"34359738368",
-        "RegisteredSealProof": 3
+	"Name":"RegisteredSealProof_StackedDrg32GiBV1",
+	"Size":"34359738368",
+	"RegisteredSealProof": 3
       },
       {
-        "Name":"RegisteredSealProof_StackedDrg64GiBV1",
-        "Size":"68719476736",
-        "RegisteredSealProof": 4
+	"Name":"RegisteredSealProof_StackedDrg64GiBV1",
+	"Size":"68719476736",
+	"RegisteredSealProof": 4
```

Comment:
Files were half formatted with tabs in some places or spaces in others. My editor did this (tabs). Choosing spaces changed other things somewhere else.